app/src/main/java/eu/kanade/presentation/theme/TachiyomiTheme.kt

```diff
@@ -63,22 +64,25 @@ private fun getThemeColorScheme(
     isAmoled: Boolean,
 ): ColorScheme {
     val colorScheme = when (appTheme) {
```

Comment:
True, I just kept it as a when because it looked neat - should probably be an if expression like this:
```kotlin
val colorScheme = if (appTheme == AppTheme.MONET) {
    MonetColorScheme(LocalContext.current)
} else {
    colorSchemes.getOrDefault(appTheme, TachiyomiColorScheme)
}
```

Reply:
Ye